lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -122,7 +123,10 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
                     course_date = course_overview.end
                 elif course_overview.has_started():
                     container_string = _("Started - {date}")
-                    course_date = enrollment_date or course_overview.dashboard_start_display
+                    if isinstance(enrollment_date, datetime.datetime):
+                      course_date = max(enrollment_date, course_overview.dashboard_start_display)
```

Comment:
`dashboard_start_display` can be a string like "Later this year"

Reply:
```
if enrollment_date and isinstance(dashboard_start_display, datetime.datetime): do the max()
else: use enrollment_date or course_overview.dashboard_start_display
```
?

Reply:
Yeah, I think that would be the most succinct version.